README.md

```diff
@@ -69,5 +70,70 @@ Usage:
 ./add_or_update.py [community.json OR official.json] [github/gitlab url OR app name [github/gitlab url OR app name [github/gitlab url OR app name ...]]]
```

Comment:
no more `community.json OR official.json`

Reply:
This PR will not remove Official.json or Community.json (not yet at least...)

Reply:
Sure, but when the new list will be there, a packager who want to add (or update) an app, have to do it on the new list, and tools will regenerate old ones (there are some code to do for that).

Reply:
A way or another, we would rather use apps.json as the main list. So I guess we should indeed modify this script as well.

Reply:
Done here https://github.com/YunoHost/apps/pull/694/commits/1c7c403dd048ae906f8458f380897f88f52ab2eb, for the PR https://github.com/YunoHost/apps/pull/694.
The current PR will not create the apps.json list yet.